src/tests/banners/bannerSelection.tsx

```diff
@@ -4,24 +4,93 @@ import {
     BannerTestSelection,
 } from '../../components/BannerTypes';
 import { AusMomentContributionsBanner } from './AusMomentContributionsBannerTest';
+import fetch from 'node-fetch';
+import { cacheAsync } from '../../lib/cache';
+
+type ReaderRevenueRegion = 'united-kingdom' | 'united-states' | 'australia' | 'rest-of-world';
+
+const readerRevenueRegionFromCountryCode = (countryCode: string): ReaderRevenueRegion => {
+    switch (countryCode) {
```

Comment:
wonder if it's worth lowercasing countryCode and matching on the lower case ? some of the ui code we looked at yesterday was using it in lower